script/package.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+set -o errexit
+set -o noclobber
+set -o nounset
+set -o pipefail
+
+if [ -z "$1" ]
+  then
+    echo "Usage: $0 package_root [ compress_type ]"
+    echo "compress_type must be one of the types accepted by dpkg-deb (-Z), default to 'xz'"
+    exit 1
+fi
+
+readonly package_root="$1"
+readonly compress_type="${2:-xz}"
+
+# calculate uncompressed size in kB, rounded up
+package_root_size="$(du --bytes --summarize --exclude=DEBIAN "${package_root}"/ | awk '{print $1}')"
+readonly package_root_size
+readonly installed_size="$(( (package_root_size + 1024 - 1) / 1024 ))"
+
+# # creates control file
+mkdir -p "${package_root}/DEBIAN/"
+cat << EOF | envsubst > "${package_root}/DEBIAN/control"
+Package: ${PACKAGE}
+Version: ${VERSION}
+Installed-Size: ${installed_size}
+Architecture: ${ARCH}
+Maintainer: ${MAINTAINER}
+Depends: ${DEPENDS}
+Description: ${DESCRIPTION}
+Homepage: https://github.com/charlie-foxtrot/RTLSDR-Airband.git
+EOF
+
+# post install script to symlink default config
+cat << EOF > "${package_root}/DEBIAN/postinst"
+#!/bin/sh
+
+set -o errexit
+
+if [ ! -e /usr/share/rtl_airband/config/default.conf ]; then
+  ln --symbolic /usr/share/rtl_airband/config/basic_emergency.conf /usr/share/rtl_airband/config/default.conf
+fi
+EOF
+
+# pre remove script to delete postinst symlink
+cat << EOF > "${package_root}/DEBIAN/prerm"
+#!/bin/sh
+
+set -o errexit
+
+if [ -L /usr/share/rtl_airband/config/default.conf ]; then
+  rm /usr/share/rtl_airband/config/default.conf
```

Comment:
If the user is creating a symlink named `default.conf` in that directory (root owned), yes it will be removed because, from the script perspective, there is no way to distinguish it from an untouched installation.

That is also the reason for the asymmetry check during installation about existence of a file instead of a symlink: to avoid overriding (or failing) if a non-script-created `default.conf` file is already there, that should be a very unlikely case since it' under package-reserved directory.

I wrote that as the `postrm` script to clean up all the installation and leave the system as before installing the package.

Either way, if the user wants to change the default file to read, I think it's best that he modifies the local systemd config instead of touching in package directory.